ppr-ui/src/utils/auth-helper.ts

```diff
@@ -64,6 +65,73 @@ export async function getStaffegisteringParty (isBcOnline: boolean): Promise<Par
   throw new Error('Auth API error getting Registering Party: status code = ' + StatusCodes.NOT_FOUND.toString())
 }
 
+// Get Account Info from from auth api /api/v1/orgs/{org_id}
+export async function getAccountInfoFromAuth (): Promise<AccountInfoIF> {
+  const url = sessionStorage.getItem('AUTH_API_URL')
+  const currentAccount = sessionStorage.getItem(SessionStorageKeys.CurrentAccount)
+  const accountId = JSON.parse(currentAccount).id
+
+  const config = { baseURL: url, headers: { Accept: 'application/json' } }
+
+  const accountInfo = await axios
+    .get(`orgs/${accountId}`, config)
+    .then(response => {
+      const data = response?.data
+      if (!data) {
+        throw new Error('Unable to obtain Registering Party from Account Information.')
+      }
+      const address: AddressIF = {
+        street: data?.mailingAddress?.street || '',
+        streetAdditional: data?.mailingAddress?.streetAdditional,
+        city: data?.mailingAddress?.city || '',
+        region: data?.mailingAddress?.region || '',
+        postalCode: data?.mailingAddress?.postalCode || '',
+        country: data?.mailingAddress?.country || ''
+      }
+
+      return {
+        id: data.id,
+        isBusinessAccount: data.isBusinessAccount,
+        name: data.businessName || data.name,
+        mailingAddress: address
```

Comment:
Does address need to be parsed out here? Would `mailingAddress: data.mailingAddress` cause an issue? 
Or is it that there is potential `null` values and you wanted string fallbacks?

Reply:
Good question. I was parsing this the same way it is currently done for another GET call to **/orgs**. But it is true, I might not need to parse here. I'll update it. Good catch!

Reply:
@cameron-freshworks updated